Voycar.Api.Web/Features/Users/Endpoints/Post/Register/Endpoint.cs

```diff
@@ -16,19 +17,22 @@ public class Endpoint : Endpoint<Request, Ok, Mapper>
     private readonly IUsers _userRepository;
     private readonly Members.Repository.IMembers _memberRepository;
     private readonly Roles.Repository.IRoles _roleRepository;
+    private readonly IPlans _planRepository;
     private readonly IEmailService _emailService;
     private readonly ILogger<Endpoint> _logger;
 
     private const string MemberRoleName = "member";
 
     public Endpoint(IUsers userRepository, Members.Repository.IMembers memberRepository, Roles.Repository.IRoles roleRepository,
-        IEmailService emailService, ILogger<Endpoint> logger)
+        IEmailService emailService, ILogger<Endpoint> logger, IPlans planRepository)
```

Comment:
Hier die gleiche Reihenfolge wie für die Felder und im Konstruktor nutzen.
```suggestion
        IPlans planRepository, IEmailService emailService, ILogger<Endpoint> logger)
```